internal/gcp/artifact_registry.go

```diff
@@ -20,15 +21,15 @@ const (
 	pageSize = 50
 )
 
-func InitArtifactRegistry() error {
+func init() {
 	ctx := context.Background()
 	service, err := registry.NewService(ctx)
 	if err != nil {
-		return fmt.Errorf("init google artifact registry: %w", err)
+		fmt.Printf("failed to initialize google artifact registry: %s\n", err)
```

Comment:
Is this actionable by the user at all? Or we'd just ideally be on the call and go figure out what's wrong.

Also was it that `init()` always runs even if the Artifact Registry service isn't needed?

Reply:
Rewrote a few things -- we fall back to just checking the regexp instead of showing this error if something goes wrong.